apps/server/src/tasks/tasks.service.ts

```diff
@@ -19,6 +20,79 @@ import { Task } from './entities/task.entity';
 
 @Injectable()
 export class TasksService {
+  @Cron('* */1 * * * *')
```

Comment:
I bet this will overlap for a few tasks, we need to prevent starting a task if a task is running. Can we do that?